Lib/AspNet/Microsoft.WindowsAzure.Storage/project.json

```diff
@@ -1,91 +1,74 @@
 ﻿{
-    "version": "7.0.1.0",
+    "version": "7.1.1.0",
     "authors": [ "Microsoft Corporation" ],
     "description": "Azure Storage SDK for CoreCLR Preview",
     "dependencies": {
         "Microsoft.Data.OData": "5.6.4",
```

Comment:
I was told today that AWS SDK works on NET Core. Maybe is time to move on, I don't want this all over again... RC2 was enough, and the Azure team learned nada.